worlds/osrs/__init__.py

```diff
@@ -0,0 +1,616 @@
+            .place_locked_item(self.create_event(ItemNames.QP_Romeo_Juliet))
+        self.multiworld.get_location(LocationNames.QP_Sheep_Shearer, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Sheep_Shearer))
+        self.multiworld.get_location(LocationNames.QP_Shield_of_Arrav, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Shield_of_Arrav))
+        self.multiworld.get_location(LocationNames.QP_Ernest_the_Chicken, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Ernest_the_Chicken))
+        self.multiworld.get_location(LocationNames.QP_Vampyre_Slayer, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Vampyre_Slayer))
+        self.multiworld.get_location(LocationNames.QP_Imp_Catcher, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Imp_Catcher))
+        self.multiworld.get_location(LocationNames.QP_Prince_Ali_Rescue, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Prince_Ali_Rescue))
+        self.multiworld.get_location(LocationNames.QP_Dorics_Quest, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Dorics_Quest))
+        self.multiworld.get_location(LocationNames.QP_Black_Knights_Fortress, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Black_Knights_Fortress))
+        self.multiworld.get_location(LocationNames.QP_Witchs_Potion, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Witchs_Potion))
+        self.multiworld.get_location(LocationNames.QP_Knights_Sword, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Knights_Sword))
+        self.multiworld.get_location(LocationNames.QP_Goblin_Diplomacy, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Goblin_Diplomacy))
+        self.multiworld.get_location(LocationNames.QP_Pirates_Treasure, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Pirates_Treasure))
+        self.multiworld.get_location(LocationNames.QP_Rune_Mysteries, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Rune_Mysteries))
+        self.multiworld.get_location(LocationNames.QP_Misthalin_Mystery, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Misthalin_Mystery)),
+        self.multiworld.get_location(LocationNames.QP_Corsair_Curse, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Corsair_Curse))
+        self.multiworld.get_location(LocationNames.QP_X_Marks_the_Spot, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_X_Marks_the_Spot))
+        self.multiworld.get_location(LocationNames.QP_Below_Ice_Mountain, self.player) \
+            .place_locked_item(self.create_event(ItemNames.QP_Below_Ice_Mountain))
```

Comment:
There's another evil `getattr` automation hack lurking here waiting to be written, but maybe not in this case, idk :D